src/useTodoState.js

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+
+export default (initialValue) => {
+  const [todos, setTodos] = useState(initialValue);
+
+  return {
+    todos,
+    addTodo: (todoText) => {
+      setTodos([...todos, todoText]);
+    },
+    deleteTodo: (todoIndex) => {
+      const newTodos = todos.filter((_, index) => index !== todoIndex);
+
+      setTodos(newTodos);
+    },
+  };
+};
```

Comment:
It's nice to have a React Context for the TODOs.
Take a read here about it: https://kentcdodds.com/blog/how-to-use-react-context-effectively

This is going to help you out in the next checkpoints (adding server, improving the app, testing, etc)